README.md

```diff
@@ -1,32 +1,51 @@
+| pylint.path             | `[]`                                                                                                                                   | "Path or command to be used by the extension to lint Python files with Pylint. Accepts an array of a single or multiple strings. If passing a command, each argument should be provided as a separate string in the array. If set to `["pylint"]`, it will use the version of Pylint available in the `PATH` environment variable. Note: Using this option may slowdown linting. <br>Examples: <br>- `"pylint.path" : ["~/global_env/pylint"]` <br>- `"pylint.path" : ["conda", "run", "-n", "lint_env", "python", "-m", "pylint"]` <br>- `"pylint.path" : ["pylint"]` <br>- `"pylint.path" : ["${interpreter}", "-m", "pylint"]`                                                                  |
+| pylint.interpreter      | `[]`                                                                                                                                   | Path to a Python executable or a command that will be used to launch the Pylint server and any subprocess. Accepts an array of a single or multiple strings.  When set to `[]`, the extension will use the path to the selected Python interpreter. If passing a command, each argument should be provided as a separate string in the array.                                                                                                                                                                                                                                              |
+| pylint.importStrategy   | `useBundled`                                                                                                                           | Defines which Pylint to be used to lint Python files. When set to `useBundled`, the extension will use the Pylint binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the Pylint binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid Pylint binary in the selected environment, it will fallback to using the Pylint binary that is shipped with the extension. This setting will be overriden if `pylint.path` is set. |
```

Comment:
```suggestion
| pylint.importStrategy   | `useBundled`                                                                                                                           | Defines which Pylint binary to be used to lint Python files. When set to `useBundled`, the extension will use the Pylint binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the Pylint binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid Pylint binary in the selected environment, it will fallback to using the Pylint binary that is shipped with the extension. This setting will be overriden if `pylint.path` is set. |
```